src/core/model/src/main/java/it/geosolutions/geostore/core/model/User.java

```diff
@@ -241,6 +242,14 @@ public void setAttribute(List<UserAttribute> attribute) {
         this.attribute = attribute;
     }
 
+    public String getIpAddress() {
```

Comment:
The model is called `IpRange` why here we named it `IpAddress`? It's a bit confusing.

Reply:
Because ipAddress represents the IP address of the user, while IPRange is the class that models the allowed range of IPs.

Reply:
I see... so this field is ment to store the user ip address only? But who stores it? The IP address should be computed dinamically, no?

Reply:
Yes, the user's IP address is computed in `RESTServiceImpl#extractAuthUser` and stored transiently in the `User` entity for later use.